apps/accounts/views.py

```diff
@@ -0,0 +1,10 @@
+from django.views.generic.edit import CreateView
+
+from .forms import UserRegistrationForm
+
+
+class UserCreateView(CreateView):
+    form_class = UserRegistrationForm
+    template_name = 'accounts/registration.html'
+    # TODO: Temporary URL.
+    success_url = '/'
```

Comment:
Maybe just add in a static thank you page to redirect to? Get it out the way - then it's a content job.